frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -391,6 +392,13 @@ export const GRAPH_DISPLAY_VALUES_SETTINGS = {
     default: "auto",
     readDependencies: ["graph.show_values"],
   },
+  "graph.max_categories": {
+    section: t`Display`,
+    title: t`Maximum number of categories`,
+    widget: "number",
+    getDefault: getDefaultMaxCategories,
+    getHidden: series => series.length < 2,
```

Comment:
It seems reasonable to make it available at least 3 series so that 2 of them can be grouped

Reply:
Same as above, we've decided to allow grouping a lone series.